src/Revamp.rei

```diff
@@ -31,7 +31,7 @@ module Compiled: {
   let count: (t, string) => int;
   let replace: (t, string => string, string) => string;
   let replaceByString: (t, string, string) => string; /* TODO: better name? */
-  let split: (t, string) => array(string); /* TODO: should be list string? */
+  let split: (t, Js.String.t) => array(option(Js.String.t)); /* TODO: should be list string? */
```

Comment:
Why `Js.String.t`? It should just be an alias of `string` anyway.

This change to `option` will need a mention in the change log (at the bottom of the README)

Reply:
As you noted, `Js.String.t` and `string` behave the same way. I have reverted it. To answer the question as to why, I don't know. I was just running through the errors until it compiled. I imagine I got an error saying the interfaces did not matched and wholesale copied the type signature in the console to the interface file. I did not take care, as, as you say, we were kinda just making it work. Also, I did not appreciate that one was an alias for the other.